board/model_manager.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import random
+import os
+
+class ModelManager:
+    def __init__(self, recommender_name, train_sampler, val_sampler, test_sampler, evaluation_metric, path_to_dataset):
+        self.recommender_name = recommender_name
+        self.train_sampler = train_sampler
+        self.val_sampler = val_sampler
+        self.test_sampler = test_sampler
+        self.evaluation_metric = evaluation_metric
+        self.path_to_dataset = path_to_dataset
+
+        self.model_id = '100xxks'
+        print("ModelManager init generated model id=", self.model_id)
+
+        
+    def sample_data_and_train(self):
+        print("sample_data_and_train called, pid =", os.getpid())
```

Comment:
nit: Can you use https://docs.python.org/2/library/logging.html#logging.Logger.debug here?